creator-cards/kosala-nuwan.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "name": "It's actually me",
+  "sectionTitle": null,
+  "countryCode": "lk",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "Currently Architect at @virtusacorp. JavaScript .NET Azure Serverless devocate. Huge GitHub nerd. Nonconformist.",
+  "tags": ".NET, JavaScript, Azure, Serverless, Devcontainers",
+  "socials": {
+    "youtube": "",
+    "linkedin": "https://www.linkedin.com/in/kosalanuwan/",
+    "twitter": "https://twitter.com/kosalanuwan",
+    "github": "https://github.com/kosalanuwan",
+    "mastodon": "https://fosstodon.org/@kosalanuwan",
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "Articles",
+      "url": "https://kosalanuwan.github.io/"
+    }
+  ],
+  "feeds": [
+    {
+      "type": "personal",
+      "url": "https://keepontruckin.hashnode.dev/rss.xml"
```

Comment:
There's some issues in this feed can you check it here please:
https://validator.w3.org/

Reply:
I am assuming the validation error we are seeing is:
```
Schema Error: XML document with no namespace; cannot determine any schema to use for validation.
https://keepontruckin.hashnode.dev/rss.xml
```
I was trying to figure out how to fix this but since the rss.xml is generated by Hashnode, I have no control over it.